scripts/install_ansible_debops_controller.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+################################################################################
+################################################################################
+#
+# https://docs.debops.org/en/master/user-guide/install.html
+#
+# This scripts installs DebOps and its dependencies on a (fresh) system.
+# It creates a dedicated DebOps user (e.g. ansible) who should then execute all DebOps-playbooks.
+# This script only installs DebOps globally , it does not initiliaze any DebOps-project.
+#
+################################################################################
+
+set -o nounset -o pipefail -o errexit
+
+# The username used by debops
+DEBOPS_ADMIN_USER="${DEBOPS_ADMIN_USER:-ansible}"
+
+echo "Install dependencies"
+apt-get -q update
+DEBIAN_FRONTEND=noninteractive apt-get \
+  --no-install-recommends -yq install \
+  iproute2 \
+  levee \
+  openssh-client \
+  python3-apt \
+  python3-distro \
+  python3-dnspython \
+  python3-future \
+  python3-ldap \
+  python3-openssl \
+  python3-pip \
+  python3-wheel \
+  python3-setuptools \
+  procps \
+  sudo \
+  tree \
+  git \
+  gpgv2 \
+  uuid-runtime \
+  sshpass \
+  encfs \
+  dirmngr \
+  rsync \
+  pass
+
+echo "Install ansible and debops"
+pip3 install \
+ debops[ansible]
```

Comment:
Hmm... This script seems to be written for use on the `root` account. What if you could rewrite it to be executed on an unprivileged account with `sudo` access (user would be asked by `sudo` for the password if needed)? That way you could install DebOps and Ansible in the user home directory instead of globally and not interfere with system Python environment. It can be done with:
```
pip3 install --user debops[ansible]
```
I imagine that you based this script on the one in the Dockerfile? :-)

Reply:
Installing in user-mode requires changing the PATH variable and so on. But it's doable:

```
## See https://docs.debops.org/en/master/user-guide/install.html#installation-of-debops-with-ansible-included
# The above command will install the debops Python package,
# as well as ansible Python package with optional dependencies used by DebOps roles.
# They will be installed in the ~/.local/lib/python2.7/site-packages/ directory,
# the scripts and other binaries will be installed
# in ~/.local/bin directory which should be included in your $PATH.
APPEND_PATH_LINE='export PATH="${PATH}:${HOME}/.local/bin"'
grep -Fxq "${APPEND_PATH_LINE}" "${HOME}/.bashrc" || echo "${APPEND_PATH_LINE}" >> "${HOME}/.bashrc"
source ~/.bashrc
```